README.md

```diff
@@ -49,6 +49,7 @@ $ cdk destroy
 | [ecs-service-with-logging](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/ecs/ecs-service-with-logging/) | Starting a container fronted by a load balancer on ECS |
 | [fargate-service-with-logging](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/ecs/fargate-service-with-logging/) | Starting a container fronted by a load balancer on Fargate |
 | [custom-logical-names](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/custom-logical-names/) | Example of how to override logical name allocation |
+| [http-proxy-apigateway](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/proxy-apigateway/) | Use ApiGateway to set up lambda proxy |
```

Comment:
As far as I can tell, this isn't setting up a lambda proxy, but a http proxy

```suggestion
| [http-proxy-apigateway](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/proxy-apigateway/) | Use ApiGateway to set up a http proxy |
```

Reply:
I agree with you, that is my typo